src/richObjectStringParser.js

```diff
@@ -91,9 +89,8 @@ export default {
 		var lastSlashPosition = parameter.path.lastIndexOf('/')
 		var firstSlashPosition = parameter.path.indexOf('/')
 		parameter.path = parameter.path.substring(firstSlashPosition === 0 ? 1 : 0, lastSlashPosition)
+		parameter.title = parameter.path.length === 0 ? '' : t('notifications', 'in {path}', parameter)
```

Comment:
changes the parameter. with `Object.assign` you can mimic the old logic and keep the parameter untouched

Reply:
right... yeah elts not change behavior